store/store.go

```diff
@@ -554,6 +554,18 @@ type requestOptions struct {
 
 // doRequest does an authenticated request to the store handling a potential macaroon refresh required if needed
 func (s *Store) doRequest(client *http.Client, reqOptions *requestOptions, user *auth.UserState) (*http.Response, error) {
+	// FIXME: find a better place for this?
```

Comment:
Thanks, I moved it over to newHTTPClient() and use that function now in download() too. I'm not sure why it was not using that before, so maybe I do something wrong here. But it looks like it makes sense, please let me know if I'm missing something.